src/components/core/InlineManager.js

```diff
@@ -66,32 +66,30 @@ export default function InlineManager( Base = class {} ) {
 
                             inline.offsetX = xoffset;
 
-                            // Workaround : Sometimes, ThreeMeshUI linebreaks before a newline "\n"
-                            //              and `lines.push([ inline ])` push the newline char "\n" itself as first char
-                            //
-                            // LetterSpacing feature was introducing a visual glitch
-                            // by adding constant letterSpacing to those hidden chars
-                            //
-                            // So as workaround if the first char has a width of 0, do not add letterSpacing
-                            if( inline.width > 0 ){
-
-                                // Do not kern first letter of a line, its has not lefthanded peer char
-                                // But still use the letterspacing
-                                return xadvance + xoffset + letterSpacing;
-                            }else{
 
-                                // When line breaker here "\n" its width is 0
-                                // Fix by setting width of 0
-                                // as letterSpacing was still adding constant offset on empty char
+                            // if the first inline of this new line :
+                            //   has a width of zero ( such as "\n" glyph )
+                            //   or
+                            //   is a space
+                            // @TODO: This might be more widely supported "\r \t"
```

Comment:
I don't have cases right now, but we can already imagine that would require further changes